src/win/getpagesize.h

```diff
@@ -0,0 +1,34 @@
+/* getpagesize emulation for systems where it cannot be done in a C macro.
+
+   Copyright (C) 2007, 2009-2012 Free Software Foundation, Inc.
```

Comment:
Can't include GPL code in an MIT licensed codebase, is this the only file that contains GPL licensed code or is there more?

Reply:
The only two files taked from the web and edited are `getpagesize.h` and `mman.h`. The former has the license specified at the beginning of the file; the latter no and so I cannot guess which type of license it has. Some similar versions can be found: https://github.com/dimichgh/mmap.js/blob/master/src/winmmap.h ,  https://github.com/ozra/mmap-io/blob/master/src/mman.h but always without license.

Reply:
I'm afraid GPL code is not allowed in MIT licensed project (such as this). Sorry!

Reply:
I'm looking for alternative implementations but being ony 4 lines of code the solution is always the same. Also on stackoverflow (https://stackoverflow.com/a/7850494) is done in the same way using different variable names. In fact it is only a call to a Windows system function as explained in the official documentation: https://msdn.microsoft.com/en-us/library/windows/desktop/ms724958%28v=vs.85%29.aspx?f=255&MSPPError=-2147217396.
How can I rewrite and include it without violating licenses?
Thanks a lot.

Reply:
I've tried to remove GPL code in the best possible way. Can you check if now it's ok here https://github.com/lucari93/mmap.js? Thanks